docs/snippets/gna/configure.py

```diff
@@ -7,12 +7,14 @@
 
 from snippets import get_model
 
-model = get_model()
 
-# TODO: no GNA properties to replace strings
-#! [ov_gna_exec_mode_hw_with_sw_fback]
-core = ov.Core()
-compiled_model = core.compile_model(
-    model, device_name="GNA", config={"GNA_DEVICE_MODE": "GNA_HW_WITH_SW_FBACK"}
-)
-#! [ov_gna_exec_mode_hw_with_sw_fback]
+def main():
```

Comment:
I suppose original snippets were written explicitly inside without main to prevent indentations in snippets..

Reply:
discussed with @jiwaszki 
dropped that commit